contrib_bots/lib/poll.py

```diff
@@ -0,0 +1,105 @@
+
+
+class PollHandler(object):
+    '''
+    This plugin facilitates polling.
+    '''
+
+    def __init__(self):
+        # polls is a dict of dict. First dict corresponds to poll title,
+        # second refers to answers in poll
+        self.polls = dict()
+
+        # streams is a convenience object that keeps track of the latest
+        # question for each stream.
+        self.streams = dict()
+
+    def usage(self):
+        return '''
+        This plugin facilitates polling.
+
+        Commands:
+        @question poll title: option 1, option 2, option 3
```

Comment:
I'd write the strings that are to be replaced in `<>` with no whitespaces, e.g. `<poll_title>`.